src/main/charts/bitbucket/values.yaml

```
@@ -76,6 +76,9 @@ serviceAccount:
   #
   imagePullSecrets: []
   # - name: secretName
+
+  #Annotations to add to the ServiceAccount if create is true
```

```suggestion
  # -- Annotations to add to the ServiceAccount (if created)
```

Please apply to all `values.yaml` files.